### DISCUSSION TITLE
Possibility of CMS producing in network files for Medicare fee schedule

### DISCUSSION THREAD

**Discussion by brbucher** [CMSgov/price-transparency-guide#523] [Ideas]
Would CMS be open to producing files for the Medicare fee schedule?

It seems like a common scenario is to have services priced at a percentage of the Medicare fee schedule.
If CMS were able to release a standard file for those prices, I think it would be much simpler for organizations producing the files to either point to the standard and report what percentage they price at for specific plans, or to import the necessary data into their own files and make the modifications.

**Comment by nsade4343** [comment]
I would also like to know if CMS will be producing the Medicare fee schedule json files, specifically for hospitals.

**Comment by jwerner15083** [comment]
It makes a lot more sense to simplify this down to a single "% of Medicare" call vs outlining every single code.  For example the professional side we have to outline EVERY single code (Lab, Anesthesia, Professional, DME, Pen, etc) with the individual rate for each one.  Makes zero sense since all we are doing is repeating the data over and over with a different rate.

**Comment by shaselton-usds** [comment]
@brbucher The Medicare fee schedules are already available online. Depending on what you need, they'll be in different locations on cms.gov. I'd suggest trying that out.

**Reply by brbucher** [reply]
While the fee schedules are available online, there are lots of variables that go into gathering the correct rate.   @sbenphia has a better description on this below.
Generating the in network files is already a complex process when dealing with data that exists within the system. 
Making calls outside the system to gather the rates available online seems like it would be prohibitive and having a standard file to reference from CMS seems like a good path forward for a standard reference rather than leaving it up to different implementations to attempt to get at the correct rates.

**Reply by jwerner15083** [reply]
@shaselton-usds Can you explain how location based feeschedule decisions need to be made?  For example we have a contract with a large provider organization who operates over multiple carriers and localities.  We do not have each individual billing NPI/location, we contract with them at a tin level.  Determining the accurate physician reimbursement for a specific code is based not just on a table, but based on the location (which is only available on the claim) and the table.  So there is no way to accurately present the data on the MRF at the provider level.  Yet we are very accurate when we have a specific claim.

There are also reimbursement options that have patient characteristics in the reimbursement (ESRD, etc).   Simply pulling in a table from CMS will not cover this either...how is that represented?

**Reply by cheernurse** [reply]
Have you tried downloading data from here for different provider type and service categories? 
https://data.cms.gov/provider-data/

What's your experience of FAIR % price based on the Medicare price? 

I'm building a consumer marketplace price search and recommendation engine. 
Just like you guys, I want to make a simple estimate of cost as starting point, so consumers don't just walk away. 
Consumer can continue the steps to communicate with providers using our estimate to ask the accurate out-of-pocket payment assuming the providers have tools to make the calculation and generate the Good Faith Estimate document for the patient. 

If you have any good ideas, please share with me.... certainly would love to build the calculator for mom-n-pop providers. 

-Wen      cheernurse@gmail.com    480-815-1628

**Comment by sbenphia** [comment]
I think it would be a daunting task to find all of the Medicare fee schedules, formulas, exceptions, and regulatory updates (some of the numbers and ratios are buried in the federal register) and create a machine readable file.  The best somebody could do is create a "Medicare-Like" machine readable file that would involve a reasonable amount of discretionary calls and some hefty disclaimers (similar to Medicare's own statements on their inpatient pricer that the rates they calculate are only estimates, and also evidenced by the fact they still can't publish an outpatient facility calculator).   In my experience Medicare is more of an instruction set and less of a schedule, like taxes, where a significant amount of rules, intent, and directives combine with variables, formulas, and adjustments to result in a number that two experts might calculate differently and both be right (or right and mostly right).  I think a few "Medicare-Like MRF" files will soon exist because some contracts are largely based on Medicare; however, the variance in these numbers will surprise analysts (in my opinion).

**Reply by jwerner15083** [reply]
Yep unfortunately these arguments are falling on deaf ears.  I petitioned early on to do the same as the percent of charges and simplify to a formula, but with no avail.  They want every code defined and published.  

The reality is that it is not entirely possible to meet their requirement.  Many codes are locality based, which is often spanned by a contract.  How do you calculate a location affected reimbursement when you do not have a location?   

And don’t get me started on outpatient!   But let’s take one of the more common, ESRD.  Weight is part of the reimbursement formula, so how do you post a mrf that requires patient specific characteristics?   You can’t.   So any company who contracts with a provider at a % of Medicare (for dialysis) simply can’t provide even a relative effort for this.  

Can’t wait to see what people spit out.
